test/e2e/infoblock.test.js

```diff
@@ -0,0 +1,20 @@
+
+const { assert, driver } = require('vl-ui-core').Test;
+const VlInfoblockPage = require('./pages/vl-infoblock.page');
+
+describe('vl-infoblock', async () => {
+    const vlInfoblockPage = new VlInfoblockPage(driver);
+
+    before(() => {
+        return vlInfoblockPage.load();
+    });
+
+    it('ik kan een custom icon kiezen', async () => {
+        const customIconBlock = await vlInfoblockPage.getCustomIconBlock();
+
+        await assert.eventually.isFalse(customIconBlock.hasAttribute('type'));
+        await assert.eventually.isTrue(customIconBlock.hasAttribute('icon'));
+        await assert.eventually.isTrue(customIconBlock.isCalendar());
```

Comment:
De code kan best uitgebreid worden zodat de vl-icon webcomponent gebruikt wordt. Alleen op attributen checken is niet (altijd) voldoende als page object om goede e2e testen te bekomen in de projecten.